data/remnant/remnant 1 introduction.txt

```diff
@@ -2435,6 +2435,60 @@ mission "Remnant: Salvage 6"
 
 
 
+mission "Remnant: Scanning Tolerances"
+	name "Deliver Data to <planet>"
+	description `Deliver data crystal to the researchers on <destination>.`
+	landing
+	deadline 15
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		not distance 0
+	to offer
+		has "license: Remnant Capital"
+		not "remnant untrusted"
+		not "Deep: Remnant 3: Revealed: done"
+		"reputation: Remnant" > 1
+		random < 30
+	on offer
+		conversation
+			`You were just in the process of disembarking from <ship> when a young Remnant jogs up to you. "Good day, Captain <first>. Sorry to interrupt, but we have an urgent delivery that needs to go to Torza on <destination>. Could you deliver this please?"`
```

Comment:
I'm generally dubious about referring to the player's ship by name like this (some names won't really work with a 'the'), but, where it is done, I think it should be consistent and I think there generally is a 'the' before it. In fact, later in this very mission, `the <ship>` is used.
```suggestion
			`You were just in the process of disembarking from the <ship> when a young Remnant jogs up to you. "Good day, Captain <first>. Sorry to interrupt, but we have an urgent delivery that needs to go to Torza on <destination>. Could you deliver this please?"`
```